### ISSUE TITLE
Claim 'vct' is missing in Type Metadata Format

### ISSUE THREAD

**Issue by jtalir** [oauth-wg/oauth-sd-jwt-vc#241]
I'm not sure why 'vct' is not (as REQUIRED) in 6.2 Type Metadata Format. Is it bug or is this some sort of implicit? It is in the  6.1. Type Metadata Example and also it is twice referenced in 6.3.5 From Type Metadata Glue Documents: "the Consumer MUST ensure that the vct claim in the credential matches the one in the Type Metadata document" and "the Consumer MUST ensure that the value of the extends property in the Type Metadata document matches that of the vct in the Type Metadata document"

**Comment by bc-pi** [comment]
This does seem like an oversight but I'll defer to @danielfett to ask if there was a reason behind the omission or if vct needs to be added to that section https://www.ietf.org/archive/id/draft-ietf-oauth-sd-jwt-vc-04.html#name-type-metadata-format

**Comment by babisRoutis** [comment]
I believe that `vct` & `vct#integrity` have to been added to the section mentioned by @bc-pi ,

Currently, examples  do include those two, and to me  are not aligned with the section.

**Comment by jtalir** [comment]
> I believe that `vct` & `vct#integrity` have to been added to the section mentioned by @bc-pi ,
> 
> Currently, examples do include those two, and to me are not aligned with the section.

I don't think so. Claim vct#integrity belongs into sd-jwt vc, not into type metadata. In the example section #type-metadata-example, there is first example with sd-jwt vc and only second example is with type metadata. In type metadata `vct` serves as "backward reference" and not as a link to be resolved.

**Comment by babisRoutis** [comment]
Dear @jtalir 

On second thought, I agree with this. 
Mainly I was missing `vct` from Type metadata.
